### PR TITLE
Improve Contributing

### PR DESCRIPTION
**Pull request by alallema**

### CODE REVIEW

**Review by curquiza — COMMENTED** [review]
Regarding the CONTRIBUTING improvement (maybe not in this PR)
- `After which don't forget to rebuild the project.` -> why?
- `The version.sh script will add and commit the modification but you till need to verify it and push it.` -> "modifications", "still" + which modifications? to every file?
- `be sure you apply [these recommandations]` -> recommendation
- Maybe we could add information about what kind of CI (publish to npm) you trigger when creating a GitHub release, like [here](https://github.com/meilisearch/meilisearch-js/blob/main/CONTRIBUTING.md#how-to-publish-the-release) "GitHub Actions will be triggered and push the package to [npm](https://www.npmjs.com/package/meilisearch)."

**Comment by curquiza** [comment]
Also, in the CONTRIBUTING, maybe you should start by publishing to NPM and then to firebase. If there is any failure in the CI, it will prevent you pushing a wrong to firebase

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -205,32 +205,44 @@ _[Read more about this](https://github.com/meilisearch/integration-guides/blob/m
...
+
+2. To update the version package in all needed files you can run the `version.sh` script with the version number in argument: `"X.X.X"`. If you want to modify it by hand you have to update the version in the following files:
 - [`package.json`](/package.json).
-- [`package.json`](/functions/packag e.json) in the functions directory.
+- [`package.json`](/functions/package.json) in the functions directory.
 - [`extension.yaml`](/extension.yaml).
 - [`version.ts`](/functions/src/version.ts).
 
-After which don't forget to rebuild the project.
+After which, don't forget to rebuild the project to check if any changes has been forgotten on the `js` files in the `functions/lib` directory.
```
Comment (brunoocasali) [inline CONTRIBUTING.md:219]:
```suggestion
After which, don't forget to rebuild the project to check if any changes have been overlooked on the `js` files in the `functions/lib` directory.
```

**Review by brunoocasali — CHANGES_REQUESTED** [review]
Hey, I have some suggestions :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -205,32 +205,44 @@ _[Read more about this](https://github.com/meilisearch/integration-guides/blob/m
...
+The `version.sh` script will update the version in all necessary files but you still need to check them:
 ```bash
 sh script/version.sh X.X.X
 ```
 
-2. Test the extension by installing it in Firestore:
-```bash
-firebase ext:install . --project=[your_project_id]
+3. Commit and push the modifications:
+```
```
Comment (curquiza) [inline CONTRIBUTING.md:227]:
```suggestion
```bash
```

**Review by curquiza — COMMENTED** [review]
Almost good to me, well done ;)
Don't wait for my approval to merge!

Also, by curiosity, why did you remove the `commit` par from the script?

**Review by mdubus — APPROVED** [review]
LGTM 🔥

**Review by brunoocasali — APPROVED** [review]
🔥 🔥

**Comment by alallema** [comment]
bors merge

**Comment by alallema** [comment]
> Also, by curiosity, why did you remove the commit par from the script?

@curquiza 
Sorry I totally forgot to answer this. I remove it because when the script commits you can not see the modification of the files there are already committed. I find it not intuitive at all because you don't know what has changed and what are committed. So it's cool but too unclear finally.

**Comment by alallema** [comment]
> By doing `git status` you will see the committed files, and also you can check the diff of the current committed changes by doing `git diff --staged`, but you're right, it's a less known command 😉

You will only see the file added with `git status` and not the ones that have been committed same for the `git diff --staged`

**Pull request merged**